fault_injection/configs/pen.global_fi.ibex.char.hardened_check_eq_unimps.cw310.yaml

```diff
@@ -37,6 +37,9 @@ test:
   which_test: "ibex_char_hardened_check_eq_unimp"
   # which_test: "ibex_char_hardened_check_eq_2_unimps"
   # which_test: "ibex_char_hardened_check_eq_3_unimps"
+  # which_test: "ibex_char_hardened_check_eq_4_unimps"
```

Comment:
The commit message title talks about OTBN but whereas it's actually about Ibex here. This should probably be corrected.

Reply:
Argh sorry, copy-and-paste error. Fixed.